docs/team/bryanhce.md

```diff
@@ -5,41 +5,65 @@ title: Ho Cheng En Bryans's Project Portfolio Page
 
 ### Project: GuestBook
 
-GuestBook is a one stop access to get all the necessary guest data.
+GuestBook is a **desktop app for managing guests in a hotel,
+optimized for use via a Command Line Interface** (CLI)
+while still having the benefits of a Graphical User Interface (GUI).
+If you are a hotel manager who can type fast, GuestBook can get your
+guest management tasks done faster than traditional GUI apps.
 The user interacts with it using a CLI, and it has a GUI created with JavaFX.
 It is written in Java, and has about 10 kLoC.
 This project is based off AddressBook - Level 3.
 
 Given below are my contributions to the project.
 
-* **New Feature**:
-  * to be added soon
+* **New Feature**: Add isRoomClean field
 
-* **Code contributed**: [RepoSense link](https://nus-cs2103-ay2223s1.github.io/tp-dashboard/?search=&sort=groupTitle&sortWithin=title&timeframe=commit&mergegroup=&groupSelect=groupByRepos&breakdown=true&checkedFileTypes=docs~functional-code~test-code~other&since=2022-09-16&tabOpen=true&tabType=authorship&tabAuthor=bryanhce&tabRepo=AY2223S1-CS2103T-W16-1%2Ftp%5Bmaster%5D&authorshipIsMergeGroup=false&authorshipFileTypes=&authorshipIsBinaryFileTypeChecked=false&authorshipIsIgnoredFilesChecked=false)
 
-* **Project management**:
-  * to be added soon
+* **New Feature**: Add markRoomsUnclean command
+  * Allows the user to set the room clean status of all the guest in the hotel to "no" in a single command.
+  * This feature greatly aids with GuestBook's efficient. Typing in one command to change all guests' room
```

Comment:
Should this be "efficiency" rather than "efficient"?

Reply:
Yes!